packages/ui/src/components/organisms/DynamicForm/components/RSJVInputAdaters/DateInputAdater/DateInputAdater.tsx

```diff
@@ -1,17 +1,47 @@
-import { DatePickerInput } from '@/components/molecules';
+import { DatePickerChangeEvent, DatePickerInput } from '@/components/molecules';
 import { RJSFInputAdapter } from '@/components/organisms/DynamicForm/components/RSJVInputAdaters/types';
 import { useCallback } from 'react';
 
-export const DateInputAdater: RJSFInputAdapter = ({ id, formData, disabled, onBlur, onChange }) => {
+export const isDateValid = (dateString: string): boolean => {
```

Comment:
Also consider how this function doesn't check if the date is valid per se. 
```suggestion
export const isValidDate = (dateString: string): boolean => {
```